aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -852,6 +852,15 @@ def _check_databases(self):
                 else:
                     self.log.info("Database %r already exists in target", dbname)
 
+    def _check_database_size(self, max_size: float):
+        dbs_size = self.source.c(
+            "SELECT SUM(pg_database_size(oid)) FROM pg_catalog.pg_database "
+            "WHERE datname NOT IN ('template1', 'template0')"
+        )[0]["sum"]
```

Comment:
This should take in account `--filtered-db`, `--skip-table`/`--with-table` and `--no-replicate-extension-tables`.

Reply:
True. Will address. Thanks.